samples/nrf9160/azure_iot_hub/Kconfig

```
@@ -14,4 +14,7 @@ module = AZURE_IOT_HUB_SAMPLE
 module-str = Azure IoT Hub sample
 source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
 
+config DEVICE_ID_USE_UUID
+	bool "Use UUID as device ID when connecting to Azure IOT hub"
```

```suggestion
	bool "Use the devices's modem UUID as device ID when connecting to Azure IOT hub"
```

since we use `CONFIG_HW_ID_LIBRARY_SOURCE_UUID`